pkg/stack/function_helpers.go

```diff
@@ -17,12 +17,14 @@
 package stack
 
 import (
+	_ "embed"
 	"fmt"
 	"path"
 	"path/filepath"
 )
 
-const DefaulMembraneVersion = "v0.12.1-rc.5"
+//go:embed membraneversion.txt
+var DefaulMembraneVersion string
```

Comment:
There is a typo fix for this on the aws deployments fixes, so I would rebase on this once it's merged to avoid conflicts:
DefaulMembraneVersion -> DefaultMembraneVersion

Reply:
my special skill is introducing spelling errors into code bases of the world ;-)

Reply:
my signature on any source file I've written is writing recieved instead of received :P.